dispatchr-ruby/app/models/item.rb

```diff
@@ -1,3 +1,4 @@
 class Item < ApplicationRecord
   has_many :stores, through: :item_stores
+  has_many :request_items, through: :request_items
```

Comment:
The "through" is only required for join tables I believe, which is why it's used in line 2. I'm not sure if it's necessary here. Can you provide some insight, @spencerprescott?